phase/analyzer_test.go

```diff
@@ -486,7 +486,7 @@ func testAnalyzer(platformAPI string) func(t *testing.T, when spec.G, it spec.S)
 
 				it("populates target metadata from the run image", func() {
 					h.AssertNil(t, previousImage.SetLabel("io.buildpacks.base.id", "id software"))
-					h.AssertNil(t, previousImage.SetOS("windows"))
+					h.AssertNil(t, previousImage.SetOS("zindows"))
```

Comment:
what's the deal with this?

Reply:
I wanted to rid ourselves of any references to the string `windows` - I mostly did it as I removed all references but decided to leave it so it was clear we don't actually support windows OS images in this repo.